src/UITests/LoadTestFile.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UITests
+{
+    /// <summary>
+    /// Start the app, open an A11yTest file, navigate to 2 views (automated checks and UIA hierarchy), spot check loaded data, exit app
+    /// </summary>
+    [TestClass]
+    public class LoadTestFile : AIWinSession
```

Comment:
This test passes but does nothing--it's just here to provide an outline of how the AIWinDriver class can be used. I'm happy to delete it for this PR if anyone feels strongly. Otherwise, it will be updated shortly as a part of this feature 🙂